entries/m/migros.ch.json

```diff
@@ -4,7 +4,7 @@
     "mfa": "allowed",
     "categories": "retail",
     "documentation": "https://account.migros.ch/help#faqPasskeyPrio",
-    "recovery": "https://account.migros.ch/help#faqPasskeyPrio",
+    "recovery": "https://account.migros.ch/help#faqPasskeyProblem",
```

Comment:
That link doesn't work for me.
![image](https://github.com/user-attachments/assets/e1c6fad9-19c2-475d-831b-932828dce909)
```
Uncaught TypeError: z.current is null
    t NextJS
_app-15c1dfebc9027a7c.js:1:1509520
```

Reply:
I'm also getting that error now